tests/algorithms/test_mce_irl.py

```diff
@@ -441,7 +449,8 @@ def test_mce_irl_reasonable_mdp(
 
         # test MCE IRL on the MDP
         mdp = ReasonablePOMDP()
-        mdp.seed(715298)
+        mdp.reset(seed=715298)
+        # mdp.seed(715298)    # NOTE: not sure this is the correct approach in this case
```

Comment:
I'm not either! But if mdp.reset is behaving normally it seems fine to go with that for this unit test and delete this.

Reply:
You are right. Removed it.